erpnext/stock/doctype/quality_inspection/quality_inspection.py

```diff
@@ -132,3 +132,20 @@ def get_purchase_item_details(doctype, name, item_code):
 			}
 			return data
 
+def make_quality_inspections(items):
+	items = json.loads(items)
+	quality_inspections = []
+	for item in items:
+		qi = frappe.new_doc("Quality Inspection")
+
+		qi.update({
```

Comment:
There's a new field we've added called "Inspection By" (with the options of Internal and External)

@neilLasrado, what should we default this value to?